### ISSUE TITLE
My training bounding box have confidence larger than one, why is that happening?

### ISSUE THREAD

**Issue by laurence-lin** [AlexeyAB/darknet#7201]
I recently solved the under-fitting problem for my small train datasets with 90 samples. 

However, there are multiple bounding box containing same object, and there are multiple bounding box have confidence larger than one. 

This is really confusing, isn't confidence score evaluated by the probability that object with bbox belong to specific class?
I've seen some others says that confidence score is evaluated based on IoU, but even in that way the confidence score shouldn't be larger than one.

This is my prediction result:

![predictions](https://user-images.githubusercontent.com/19323465/103402385-f18ab680-4b87-11eb-8ca5-84ffe8788280.jpg)

Thank you for any help!

**Comment by stephanecharette** [comment]
When did you clone the project?  There was a bug earlier in December that would cause probability > 1.0.  There were multiple fixes, I think the last one was checked in on December 14.  Was your neural network trained with a version of darknet prior to then?

**Comment by laurence-lin** [comment]
> When did you clone the project? There was a bug earlier in December that would cause probability > 1.0. There were multiple fixes, I think the last one was checked in on December 14. Was your neural network trained with a version of darknet prior to then?

Yes, I clone the project about the start of December. So I should clone the  project again right? I would try it! Thank you!
